### PR TITLE
Fix: Remove scss

### LINKED ISSUE #201: Finish SCSS clean up
## Description
We previously converted the app to use Tailwind instead of SCSS; however, some SCSS files remain. This issue is to complete the removal of all of the SCSS files and convert any remaining dependencies (if any) to Tailwind. 

## Acceptance Criteria
- [ ] All SCSS files are removed & app still looks and works as it should.

### PR DESCRIPTION
**Pull request by denviljclarke**
### Description

This PR removes all of the SCSS files that were left & replaces the styling with tailwind. 

### Other changes

- Correct disconnected state on the proposal page (show proposal statuses of finished proposals without needing to connect
- Add proper loading skeleton for proposal actions section ( vote, queue & execute card )
- Correctly display the disconnected state on `/my-voting-power` page
- Fix full progress bar spacing in the proposals list on the home page
- Fix loading state behavior + add loading skeleton for proposals list
- Fix the loading state when fetching a proposal
- Add default value for total supply (reduce layout shift when loading)

### Tested

Ran the app locally & on preview

### Related issues

- Fixes #201

### CODE REVIEW

File: src/components/proposals-list/proposals-list.component.tsx [hunk 1]
```diff
@@ -47,130 +49,236 @@ const ProposalsTable = () => {
...
+      {headers.map((text, index) => (
         <div
           key={index}
-          className="relative grid grid-cols-[60%_40%] items-start pb-[8px] pt-[20px] font-medium first:pt-0 last:mb-0"
+          className={`overflow-hidden border-none font-inter text-base font-medium tracking-tighter ${
+            index === 0
+              ? "text-left"
+              : index === headers.length - 1
+                ? "text-right"
+                : "text-center"
```
Comment (Andrew718PLTS) [inline src/components/proposals-list/proposals-list.component.tsx:260]: Looks more complex than it is) I'd suggest to create an additional function with corresponding name where it's written like this to increase readability:
```
function getTextPosition(index, headers) {
  if (index === 0) {
    return "text-left";
  } 
  if (index === headers.length - 1) {
    return "text-right";
  }
  return "text-center";
}
```

File: src/lib/contracts/useTokens.ts [hunk 2]
```diff
@@ -118,7 +118,7 @@ export const useTokens = () => {
         mentoContractData: {
           decimals: 18,
           symbol: "MENTO",
-          totalSupply: 0n,
+          totalSupply: 610839491208273437600000000n,
```
Comment (Andrew718PLTS) [inline src/lib/contracts/useTokens.ts:121]: Is a debug value or could you tell me please what change reason?

**Review by RyRy79261 — COMMENTED** [review]
Looks good, the My Voting power page is stuck on loading the whole time. I do like @Andrew718PLTS s suggestion for the index logic

File: src/components/proposals-list/proposals-list.component.tsx (hunk 1 above)
Comment (denviljclarke) [reply src/components/proposals-list/proposals-list.component.tsx:260]: Nice one

**Review by RyRy79261 — APPROVED** [review]

**Review by Andrew718PLTS — APPROVED** [review]

**Comment by Andrew718PLTS** [comment]
@denviljclarke Approve here, please check conflicts before merging

**Pull request merged**